src/Resources/QueueMonitorResource.php

```diff
@@ -72,6 +70,21 @@ public static function getRelations(): array
         ];
     }
 
+    protected static function getNavigationGroup(): ?string
+    {
+        return config('filament-jobs-monitor.labels.navigation_group');
```

Comment:
According to new config array :

```suggestion
        return config('filament-jobs-monitor.navigation.group_label');
```